src/invert/laplace/impls/multigrid/multigrid_solver.cxx

```diff
@@ -764,16 +717,4 @@ MultigridSerial::MultigridSerial(int level, int gx, int gz, int UNUSED(px),
       output<<i<<" Ser glo dim "<<gnx[i]<<","<<gnz[i]<<endl;
     }    
   }
-
-  // This could use a Matrix
```

Comment:
Where is this allocation now being done? I see matmg is deleted in the destructor, but where is it created?

Reply:
Most of this code was done in the `MultigridAlg` constructor, which was called first, then repeated here